app/utils/ClaimToAdjustUtils.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import auth.MtdItUser
+import config.featureswitch.FeatureSwitching
+import controllers.routes
+import models.admin.AdjustPaymentsOnAccount
+import play.api.mvc.Result
+import play.api.mvc.Results.Redirect
+
+import scala.concurrent.Future
+
```

Comment:
:thinking: - as I undertand our intention for ClaimToAdjust/POA work would be to use this trait across most of the controllers?

Reply:
Yes Pavel, I've implemented this one the other POA pages as well. It cuts down on duplicated code. We had a similar thing with WithIncomeSourceFS